src/System.Runtime.Loader/tests/DefaultContext/DefaultLoadContextTest.cs

```diff
@@ -59,23 +59,38 @@ public static void LoadInDefaultContext()
             Init();
 
            // This will attempt to load an assembly, by path, in the Default Load context via the Resolving event
-           var assemblyName = "System.Runtime.Loader.Noop.Assembly";
+           var assemblyNameStr = "System.Runtime.Loader.Noop.Assembly";
 
             AssemblyLoadContext.Default.Resolving += ResolveAssembly;
```

Comment:
Should there be an attempt to load it before the Resolving handler is hooked up, to verify that it fails without the custom loading logic?